hack/examples/overlays/clusters/aws/cilium/crs/kustomization.yaml.tmpl

```diff
@@ -20,3 +20,6 @@ patches:
   - target:
       kind: Cluster
     path: ../../../../../patches/aws/crs-strategy.yaml
+  - target:
```

Comment:
Intentionally leaving this off for Docker clusters to continue testing kube-proxy there.